akka-stream/src/main/boilerplate/akka/stream/impl/GenJunctions.scala.template

```diff
@@ -33,4 +33,24 @@ private[akka] object GenJunctions {
   }#
   ]
 
+  sealed trait UnzipWithModule {
+      /** Allows hiding the boilerplate Props creation from the materializer */
+      def props(settings: ActorFlowMaterializerSettings): Props
+    }
```

Comment:
these three lines are indented by 2 space too many, please remove (boilerplate files are not automatically formatted)